components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal.tsx

```diff
@@ -54,23 +55,20 @@ const modalMessages = defineMessages({
 });
 
 const KeyboardShortcutsModal: React.FC = () => {
-    const [show, updateShow] = useState<boolean>();
+    const [show, updateShow] = useState<boolean>(false);
 
     useEffect(() => {
+        //toggles the state of shortcut dialog
+        function handleToggle(): void {
+            updateShow(!show);
+        }
```

Comment:
why is this written as function declaration? It would be way shorter with a function expression like this:

```suggestion
        //toggles the state of shortcut dialog
        const handleToggle = (): void => updateShow(!show);
```